xtras/src/send_next.rs

```diff
@@ -0,0 +1,51 @@
+use async_trait::async_trait;
+use tracing::instrument;
+use xtra::message_channel::MessageChannel;
+use xtra::refcount::RefCounter;
+
+/// Arbitrarily hard number allowing flexibility of prioritising messages in actors
```

Comment:
❓ Did you mean high?

```suggestion
/// Arbitrarily high number allowing flexibility of prioritising messages in actors
```